app/models/item.rb

```diff
@@ -21,7 +21,8 @@ def subtotal(order_id)
   end
 
   def high_bid
-    bids.maximum('price')
+    high_bid = bids.maximum('price')
+    high_bid ? high_bid : 0.00
```

Comment:
should we just use an if statement instead of ternary? this can be easier to read even when it's more lines